Workers/GameLogic/Program.cs

```diff
@@ -105,32 +102,6 @@ private static async Task RunAsync(IWorkerOptions options)
 
         private static void ProcessOpList(OpList opList)
         {
-            foreach (var logOp in opList.OfOpType<LogMessageOp>())
```

Comment:
They're deprecated anyway, just kill 'em

Reply:
Wait `LogMessageOp` is deprecated! Brb going to remove this in https://github.com/spatialos/gdk-for-unity too